posts/mixtral/index.md

```diff
@@ -0,0 +1,221 @@
+---
+title: "Build a Mixtral Chatbot with Panel"
+date: "2023-12-12"
+description: "With Transformers and llama.cpp"
+author:
+  - Andrew Huang
+  - Philipp Rudiger
+  - Sophia Yang
+categories: [showcase, panel]
```

Comment:
You could add a new category like `ai`, `llm`, `chatbot`, etc.

Reply:
good idea thanks!